index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/936703f17d.js" crossorigin="anonymous"></script>
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="header">
+            <div class="logo_menu">
```

Comment:
- [ ] **[OPTIONAL]** In your codes, you have used too many `div` tags which makes the codes hard to read 😨 Please it would be better to use the `div` tag to group various tags of HTML so that sections can be created and styles can be applied to them together. 

Kindly try to reduce them in your codes for HTML semantic reasons 👍 :+1:

For example:
```
      <div class="name"><h1>Hey There. <br/> I'm Mahdi.</h1></div>
        <div class="slogan"><h2>I am a Software Developer</h2></div>
        <div class="desc">
            <p>
            I can help you build a product , feature
             <br>or website Look through some of my 
             <br>work and experience! If you like what you
             <br> see and have a project you need coded, 
             <br> don’t hesitate to contact me.
           </p>
        </div>
```
can be modified to 👇 
```
       <h1 class="name">Hey There. <br/> I'm Mahdi.</h1>
        <h2 class="slogan">I am a Software Developer</h2>
        <p class="desc">
            I can help you build a product , feature
             <br>or website Look through some of my 
             <br>work and experience! If you like what you
             <br> see and have a project you need coded, 
             <br> don’t hesitate to contact me.
          </p>
```
And this could give the same result 😎😎